Vegeting/Screens/CreateGroup/Components/NumberOfGroupPeopleView.swift

```diff
@@ -29,6 +29,8 @@ final class NumberOfGroupPeopleView: UIView {
     
     private let numberList: [String] = ["2명", "3명", "4명", "5명", "6명", "7명", "8명"]
     weak var delegate: NumberOfGroupPeopleViewDelegate?
+    private var selectedNumber: Int?
+    
```

Comment:
selectedNumber를 별도로 저장하는 형태를 사용하지 않아도 가능합니다~!
```suggestion
func selectAlreadyNumberOfPeople(maximumNumber: Int) {
        guard let index = numberList.firstIndex(of: "\(maximumNumber)명") else { return }
        let indexPath = IndexPath(item: index, section: 0)
        numberCollectionView.selectItem(at: indexPath, animated: true, scrollPosition: .bottom)
    }
    
```

Reply:
[ee60d72](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/164/commits/ee60d72c8868fc88a065c29f3cd72b9c42ffb49e)에서 반영 완료!